Fix:
error: implicit declaration of function 'glXQueryVersion' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- m4/ax_check_gl.m4.orig	2021-08-13 23:18:24.000000000 -0500
+++ m4/ax_check_gl.m4	2023-03-22 00:15:02.000000000 -0500
@@ -80,11 +80,11 @@
 #   include <windows.h>
 # endif
 # ifdef HAVE_GL_GL_H
-#   include <GL/gl.h>
+#   include <GL/glx.h>
 # elif defined(HAVE_OPENGL_GL_H)
-#   include <OpenGL/gl.h>
+#   include <OpenGL/glx.h>
 # else
-#   error no gl.h
+#   error no glx.h
 # endif]],
                            [[glXQueryVersion(0, 0, 0)]])])
 
